src/Elmish.WPF/Program.fs

```diff
@@ -5,9 +5,9 @@ open System.Windows
 open Elmish
 
 
-/// Starts the Elmish dispatch loop, setting the bindings as the DataContext
-/// for the specified FrameworkElement. Non-blocking. This is a low-level function;
-/// for normal usage, see runWindow and runWindowWithConfig.
+/// Starts an Elmish dispatch loop, setting the bindings as the DataContext for the
+/// specified FrameworkElement. Non-blocking. If you have an explicit entry point where
+/// you control app/window instantiation, runWindowWithConfig may be a better option.
```

Comment:
I prefer "might be" over "may be" since "may be" somewhat looks and definitely sounds like "maybe".
```suggestion
/// you control app/window instantiation, runWindowWithConfig might be a better option.
```